### ISSUE TITLE
Fix support for using PNGs

### ISSUE THREAD

**Issue by AnthonyvW** [UMN-LATIS/LinearStitch#17]
When using a folder full of png files, LinearStitch will fail to find the files despite it being supported in stitcherGUI.py.

**Comment by cmcfadden** [comment]
Good call. Started looking at this, just leaving a few notes to myself:

* need to investigate whether we should skip pruning for non-jpeg sources, since that depends on jpeg file sizes?
* refactor all our hardcoded jpeg references to an array of formats
* refactor from shell-based wildcard expansion to using the `glob` python package

**Comment by AnthonyvW** [comment]
Is there a consensus on how pruning should be handled for non-jpeg sources? I'm working on adding in PNG support.

**Comment by cmcfadden** [comment]
Easiest answer is just to skip it for PNGs because you can’t cheat it with file size. 

It might be worth testing the perf impact of actually opening each file and computing a laplacian value though - you could do it for the whole stack then just prune outliers. We have that code in [linearsnap](https://github.com/UMN-LATIS/linear-snap-gui) but it runs on much smaller frame sizes so I’m not sure what the cost would be on a 50mp frame.
